target/linux/ath79/dts/qca9563_tplink_archer-c7-v5.dts

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "qca9563_tplink_archer-x7-v5.dtsi"
+
+/ {
+	compatible = "tplink,archer-c7-v5", "qca,qca9563";
+	model = "TP-Link Archer C7 v5";
+};
+
+&mtdparts {
+	factory-uboot@0 {
+		label = "factory-uboot";
+		reg = <0x000000 0x020000>;
+		read-only;
+	};
+
+	uboot: u-boot@20000 {
```

Comment:
same as above.

We also do not need the "uboot label for the node as only the `info` and `art` partition are referenced in the included dtsi.